rotkehlchen/accounting/constants.py

```diff
@@ -1,206 +1,264 @@
     ),
     EventCategory.APPROVAL: EventCategoryDetails(
         label='approval',
         icon='mdi-lock-open-outline',
+        direction=EventDirection.NEUTRAL,
     ),
     EventCategory.DEPOSIT: EventCategoryDetails(
         label='deposit',
         icon='mdi-arrow-expand-up',
         color='green',
+        direction=EventDirection.OUT,
     ),
     EventCategory.WITHDRAW: EventCategoryDetails(
         label='withdraw',
         icon='mdi-arrow-expand-down',
+        direction=EventDirection.IN,
     ),
     EventCategory.AIRDROP: EventCategoryDetails(
         label='airdrop',
         icon='mdi-airballoon-outline',
+        direction=EventDirection.IN,
     ),
     EventCategory.BORROW: EventCategoryDetails(
         label='borrow',
         icon='mdi-hand-coin-outline',
+        direction=EventDirection.IN,
     ),
     EventCategory.REPAY: EventCategoryDetails(
         label='repay',
         icon='mdi-history',
+        direction=EventDirection.OUT,
     ),
     EventCategory.DEPLOY: EventCategoryDetails(
         label='deploy',
         icon='mdi-swap-horizontal',
+        direction=EventDirection.NEUTRAL,
     ),
     EventCategory.BRIDGE_IN: EventCategoryDetails(
         label='bridge',
         icon='mdi-arrow-expand-up',
         color='red',
+        direction=EventDirection.OUT,
     ),
     EventCategory.BRIDGE_OUT: EventCategoryDetails(
         label='bridge',
         icon='mdi-arrow-expand-down',
         color='green',
+        direction=EventDirection.IN,
     ),
     EventCategory.GOVERNANCE: EventCategoryDetails(
         label='governance',
         icon='mdi-bank',
+        direction=EventDirection.NEUTRAL,
     ),
     EventCategory.DONATE: EventCategoryDetails(
         label='donate',
         icon='mdi-hand-heart-outline',
+        direction=EventDirection.OUT,
     ),
     EventCategory.RECEIVE_DONATION: EventCategoryDetails(
         label='receive donation',
         icon='mdi-hand-heart-outline',
+        direction=EventDirection.IN,
     ),
     EventCategory.RENEW: EventCategoryDetails(
         label='renew',
         icon='mdi-calendar-refresh',
+        direction=EventDirection.OUT,
     ),
     EventCategory.PLACE_ORDER: EventCategoryDetails(
         label='place order',
         icon='mdi-briefcase-arrow-up-down',
+        direction=EventDirection.NEUTRAL,
     ),
     EventCategory.TRANSFER: EventCategoryDetails(
         label='transfer',
         icon='mdi-swap-horizontal',
+        direction=EventDirection.IN,
     ),
     EventCategory.STAKING_REWARD: EventCategoryDetails(
         label='staking reward',
         icon='mdi-treasure-chest',
+        direction=EventDirection.IN,
     ),
     EventCategory.CLAIM_REWARD: EventCategoryDetails(
         label='claim reward',
         icon='mdi-gift',
+        direction=EventDirection.IN,
     ),
     EventCategory.LIQUIDATE: EventCategoryDetails(
         label='liquidate',
         icon='mdi-water',
+        direction=EventDirection.OUT,
```

Comment:
Notes:

1. Specify. GOT_LIQUIDATED. So it's a loss.
2. As mentioned above EventCategory.LIQUIDATOR_REWARDS or something like this needs to be created for the `IN` counterpart. Need to figure out icons and stuff.

Reply:
Resolved with suggested change.